casper/src/main/scala/coop/rchain/casper/util/rholang/InterpreterUtil.scala

```diff
@@ -61,15 +63,15 @@ object InterpreterUtil {
   ): F[BlockProcessing[Option[StateHash]]] = {
     val incomingPreStateHash = ProtoUtil.preStateHash(block)
     for {
-      _                  <- Span[F].mark("before-unsafe-get-parents")
-      parents            <- ProtoUtil.getParents(block)
-      _                  <- Span[F].mark("before-compute-parents-post-state")
-      preStateHashEither <- computeParentsPostState(parents, s, runtimeManager).attempt
-      _                  <- Log[F].info(s"Computed parents post state for ${PrettyPrinter.buildString(block)}.")
-      result <- preStateHashEither match {
+      _                   <- Span[F].mark("before-unsafe-get-parents")
+      parents             <- ProtoUtil.getParents(block)
+      _                   <- Span[F].mark("before-compute-parents-post-state")
+      computedParentsInfo <- computeParentsPostState(parents, s, runtimeManager).attempt
```

Comment:
I'd better call this `preStateComputed`, which should include preStateHash and deploys rejected when merging. And maybe make a `PreState` case class to describe result of computing of preState for next state transition. But I'm ok with this, thought would be nice to make code more verbose.

Reply:
@tgrospic @nzpr Get consensus. I am not really good at naming. Actually now it is the right time for me to vote to get a consensus in a three-node(person) network. Because i am lazy and don't want to change, i vote for `computedParentsInfo`.